backend/config.default.yaml

```diff
@@ -65,3 +65,7 @@ migration:
 
 feature-flag:
   disabled-authz: true
+
+cron_job:
+  update_parent_timestamp:
+    frequency: 1 #frequency to update parent timestamp in minute, default 1
```

Comment:
Can we make the config be seconds by default? It will give us more flexibility.

Reply:
Change config with the second and LIMIT where default value of the same is 60s & 100 respectively.